src/components/va-alert/test/va-alert.e2e.ts

```diff
@@ -140,4 +139,15 @@ describe('va-alert', () => {
     expect(analyticsSpy).toHaveReceivedEventTimes(0);
   });
 
+  it('has the correct accessibility attributes when in an error state', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<va-alert status="error"><h4 slot="headline">This is an alert</h4><div>This is the alert content</div>',
+    );
+
+    const element = await page.find('va-alert >>> .alert');
+
+    expect(element).toEqualAttribute('role', 'alert');
```

Comment:
What do you think about adding a line to check for the `aria-live="assertive"` too? I know it's pedantic and picky, so add it if you feel compelled, ignore if not.

```javascript
expect(element).toEqualAttribute('aria-live', 'assertive');
```

Reply:
You're right, I actually had that in before I was testing another change. I'll add it back in